### PR TITLE
feat: Add Single Video Page

### PR DESCRIPTION
**Pull request by Divya70**
feat: Add single video page

### CODE REVIEW

File: src/components/SingleVideoCom/singlevideocom.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      height: 100vh;
+}
+.single-video-main-container{
+      width: 70%;
+      height: 35rem;
+      box-shadow: 1px 6px 6px 7px rgba(0,0,0,0.75);
+      
+}
+.image-cont {
+      /* width: 100%; */
```
Comment (znjs) [inline src/components/SingleVideoCom/singlevideocom.css:12]: can remove the commented code

File: src/pages/SingleVideoPage/SingleVideoPage.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { React, useEffect } from "react";
+import axios from "axios";
+import "./singlevideopage.css";
+import { SingleVideoCom } from "../../components/SingleVideoCom/SingleVideoCom";
```
Comment (znjs) [inline src/pages/SingleVideoPage/SingleVideoPage.js:4]: can make an `index.js` file for components and use barrel export

**Review by znjs — COMMENTED** [review]
- Good PR
- Good naming conventions
- easy to understand code

File: src/components/SingleVideoCom/singlevideocom.css (hunk 1 above)
Comment (ScoobyDubeyDo) [inline src/components/SingleVideoCom/singlevideocom.css:12]: **suggestion**: Remove comments if not in use

**Review by ScoobyDubeyDo — APPROVED** [review]

File: src/components/SingleVideoCom/singlevideocom.css (hunk 1 above)
Comment (Divya70) [reply src/components/SingleVideoCom/singlevideocom.css:12]: sure i will remove that

File: src/pages/SingleVideoPage/SingleVideoPage.js (hunk 2 above)
Comment (Divya70) [reply src/pages/SingleVideoPage/SingleVideoPage.js:4]: okay , i will do that in next pr

**Review by znjs — APPROVED** [review]

**Pull request merged**